docs/src/ToricGeometry/NormalToricVarieties.md

```diff
@@ -0,0 +1,49 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["tg_ntv.md"]
```

Comment:
Do these filename need to be adapted?

Reply:
Probably. What is the convention? Lower snail case and filename=Pages? I was a little confused when I grepped for `Pages`.

Reply:
when we renamed various files in `docs`, I forgot to adjust these `Pages` entries, so there are many incorrect examples now. I'll submit a fix